packages/components/src/overlays/Popover/tests/chromatic/Popover.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button, Heading, Footer, Link, ButtonGroup, Stack, Text } from "@hopper-ui/components";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Popover, PopoverTrigger } from "../../src/Popover.tsx";
+
+const meta = {
+    title: "Components/Popover",
+    component: Popover
+} satisfies Meta<typeof Popover>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const TRIGGER = "Trigger";
+const TITLE = "Engagement score";
+const CONTENT = "The engagement score is the weighted average of the key metric scores below. To calculate these, we take the latest answers from each survey questions within the last 90 days.";
+const LINK = "Tell me more";
+const PRIMARY_ACTION = "Got it";
+const SECONDARY_ACTION = "Next";
+
+export const Dev = {
+    render: args => (
+        <PopoverTrigger>
+            <Button>{TRIGGER}</Button>
+            <Popover {...args} >
+                <Text>{TITLE}</Text>
+                <p>{CONTENT}</p>
+                <Footer>
+                    <Link href="#">{LINK}</Link>
+                </Footer>
+            </Popover>
+        </PopoverTrigger>
+    )
+} satisfies Story;
+
+export const Default = {
+    render: args => (
```

Comment:
If feels like something is missing here. We are taking screenshto of blue buttons, without the popover opened. Is it possible to add the open={true} property so we take screenshots of opened popovers to be able to do regression testing on the popover's content